README.md

```diff
@@ -2,7 +2,7 @@
 
 # SDR Integration Tests
 
-This script drives a browser to do inter-system integration testing of SDR in the staging environment.
+This script drives a browser to do inter-system integration testing of SDR in the staging or QA environment.
```

Comment:
where does it say "how to run in qa environment" ?   I assume just having the settings file isn't enough -- I would presumably have both of them at some point from doing my work ...

Reply:
In the Run Tests / SDR Environments section.

Reply:
Does the fact that i missed it imply it should go higher?   Once I have my configs set up, I'm gonna want this info handier.

Reply:
@ndushay how about I do a follow-up PR focused on re-organizing the README?

Reply:
sure - sounds good.  Not even that urgent.